src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -96,26 +97,25 @@ public static void validateTaskConfig(final PluginTask task)
      * For getting the timestamp value of the node
      * Sometime if the parser could not parse the value then return null
      * */
-    private static Timestamp getTimestampValue(PluginTask task, Column column, String value)
+    private static Timestamp getTimestampValue(final PluginTask task, final Column column, final String value)
     {
-        List<ColumnConfig> columnConfigs = task.getColumns().getColumns();
+        final List<ColumnConfig> columnConfigs = task.getColumns().getColumns();
         String pattern = DEFAULT_TIMESTAMP_PATTERN;
-        for (ColumnConfig config : columnConfigs) {
+        for (final ColumnConfig config : columnConfigs) {
+            final ConfigSource columnConfig = config.getConfigSource();
```

Comment:
variable names here sound confusing. Would suggest to change:

```
for (final ColumnConfig columnConfig : columnConfigs) {
            final ConfigSource columnConfigSource = config.getConfigSource();
```

Or something like that to show `column config` and `config source`

Reply:
Done